mesh_segmenter/src/convex_hull.cpp

```diff
@@ -0,0 +1,83 @@
+#include <mesh_segmenter/convex_hull.h>
+
+ConvexHullGenerator::ConvexHullGenerator(){}  
+
+void ConvexHullGenerator::makeMesh(const std::string& input, pcl::PointCloud<pcl::PointXYZ>& inMesh)
+{
+  //expects a .ply file
```

Comment:
Reading files should have some kind of error checking to insure the file exists, and that it is read successfullly. Perhaps returning a bool, generating some error output etc